archlinux/issues.md

```diff
@@ -0,0 +1 @@
+https://gitlab.archlinux.org/archlinux/archlinux-docker/issues
```

Comment:
Should we wait to merge this until this is something users can actually contribute to, or is this containing at least some information a net win over the current unusable link? :sweat_smile:

Reply:
Actually, yes! The current issue tracker is closed AND contains no issues. The GitLab link is just closed.